adam/core/transforms.py

```diff
@@ -0,0 +1,126 @@
+import abc
+from typing import TypeVar
+
+import numpy as np
+
+T = TypeVar("T")
+
+
+class AbstractTranforms(abc.ABC):
+    def __init__(self) -> None:
+        pass
+
+    def R_from_axisAngle(self, axis, q):
+        [cq, sq] = [np.cos(q), np.sin(q)]
```

Comment:
```suggestion
        cq, sq = np.cos(q), np.sin(q)
```

no need for brackets

Reply:
Other nit: stick with sneak case letters without mixing with camel case: `R_from_axis_angle`